src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -63,10 +64,7 @@ public function load(array $configs, ContainerBuilder $container)
         $config = $this->processConfiguration($configuration, $configs);
 
         // image-formats
-        $container->setParameter(
-            'sulu_media.image_format_file',
-            $container->getParameterBag()->resolveValue($config['image_format_file'])
```

Comment:
How could you resolve this? Doesn't the file paths contain any parameter variables?